### ISSUE TITLE
Any way to add an option to pin the sidebar?

### ISSUE THREAD

**Issue by driftywinds** [ranmaru22/firefox-vertical-tabs#69]
For me personally, I do not like the fact that I have to hover over the condensed (when it only shows the favicons) sidebar to fully open it up.
Is there a way you can add a toggle for keeping the sidebar non-condensed all the time, the width of which can be set by dragging it left or right?

**Comment by ranmaru22** [comment]
Yes, you can literally just delete most of the code inside the `userChrome.css` file. All the stuff inside there either hides the original tab bar or defines this hover behavior. If you delete all of that, you'll be left with Firefox's normal sidebar. That one you can resize as you will. 

The actual styling of the tabs comes from the `tabCenterReborn.css` file. You will need to adjust the hover effects there too though if you want consistent behavior (otherwise you'll have "open/close animations" without anything opening or closing).

**Comment by driftywinds** [comment]
> Yes, you can literally just delete most of the code inside the `userChrome.css` file. All the stuff inside there either hides the original tab bar or defines this hover behavior. If you delete all of that, you'll be left with Firefox's normal sidebar. That one you can resize as you will.
> 
> The actual styling of the tabs comes from the `tabCenterReborn.css` file. You will need to adjust the hover effects there too though if you want consistent behavior (otherwise you'll have "open/close animations" without anything opening or closing).
Thank you for your reply, but I'm not a dev so I don't know what to delete and what not. Is there anyway to help me out with that so that I don't accidentally break things and have to learn by trial and error?

**Comment by ranmaru22** [comment]
Sure. You can straight up delete everything from line 238 until end of file. Everything above that is customizing the top (so hiding the tab bar and adjusting window decorations) and everything below is the sidebar which you don't need. 

That will revert the sidebar back to the normal behavior.

**Comment by josiah-adfhaiti** [comment]
I deleted everything below line 238 and added this ↓

`#sidebar-box[sidebarcommand*="tabcenter"] #sidebar-header {
    visibility: collapse;
    display: none;
}`

to get rid of the Tab Center Reborn header

**Comment by driftywinds** [comment]
For people looking at this in the future - line 238 used to be the start of the "General Styles" part of the code, and that's what you need to remove in the latest version of the code.
